idflakies-legacy/src/main/java/edu/illinois/cs/dt/tools/detection/LegacyDetectorPathManager.java

```diff
@@ -0,0 +1,128 @@
+package edu.illinois.cs.dt.tools.detection;
+
+import com.google.common.base.Preconditions;
+
+import edu.illinois.cs.dt.tools.utility.Level;
+import edu.illinois.cs.dt.tools.utility.Logger;
+import edu.illinois.cs.dt.tools.utility.PathManager;
+
+import edu.illinois.cs.testrunner.configuration.Configuration;
+import edu.illinois.cs.testrunner.coreplugin.TestPluginUtil;
+import edu.illinois.cs.testrunner.util.ProjectWrapper;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class LegacyDetectorPathManager extends PathManager {
+    
+    @Override
+    public Path detectionResultsInstance() {
+        return path(DETECTION_RESULTS);
```

Comment:
Let's have it call the instance versions (from this same class) instead of redirecting towards the static methods.